runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/SdkComponents.java

```diff
@@ -304,7 +304,7 @@ public String registerEnvironment(Environment env) {
     if (existing != null) {
       environmentId = existing;
     } else {
-      String name = uniqify(env.getUrn() + "_", environmentIds.values());
```

Comment:
Removing this from here, adding to uniqify would be a proper place, but I am not sure whether any other logic that uses uniqify assumes current pattern in tests etc... so leaving as is.